benchmark_test/scripts/benchmark-test-setup.sh

```diff
@@ -201,14 +205,16 @@ main(){
 
   run_fio_local_path_test
 
-  adjust_replica_count 1
-  run_fio_longhorn_test 1
+  adjust_replica_count 3
+  run_fio_longhorn_test 3
 
   adjust_replica_count 2
+  kubectl cordon "$(kubectl get nodes | awk 'NR!=1 && $3!~/control-plane/ {print $1}' | awk 'NR==3')"
```

Comment:
This is based on the assumption we have 3 worker nodes.

Reply:
Yes. It's temporarily OK since there's no parameter to change the worker count in the current Jenkins longhorn-benchmark-test pipeline, so the worker count is temporarily always 3.